sunflower-land/sunflower-land
# [FIX] Fix default selected item in inventory. Add cloud gazer.

# Description

This is PR with a bunch of small fixes. 

- Align the selected item between the inventory modal and the quick select. https://discord.com/channels/880987707214544966/944746072247504936/955096902666436760 
- Replace the Land Extension NPC with a little cloud gazer gif (see vid)
- Fix inventory modal to not show the item display panel if there if the user has no items.
- Fix tailor by adding highlight, pointer and creating clickable image when not readonly

## Type of change

Please delete options that are not relevant.


# How Has This Been Tested?

- Open inventory modal. Selected item should be the same at the current selected item in the quick select.
- Remove all items from inventory. There shouldn't be an item detail panel visible.
- Check tailor to make sure you can hover and click the image when on your own farm. Go read only and make sure you can't click the tailor

<img width="510" alt="Screen Shot 2022-03-22 at 2 49 19 pm" src="https://user-images.githubusercontent.com/17863697/159404233-0626d9fc-e1c7-49a4-99f8-f5a23413cdfc.png">

https://user-images.githubusercontent.com/17863697/159403742-f43bb406-8f0c-465a-aee3-62525dd29bc9.mov



# Checklist:
